### PR TITLE
Adding metriq qedc blog post.

### PR DESCRIPTION
**Pull request by vprusso**
Adding blog post for Metriq <-> QED-C pipeline.

### CODE REVIEW

File: posts/_markdown/metriq_qedc.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+makes transparent, accessible benchmarks available to everyone in the quantum
+computing community.
+
+The Technical Advisory Committee (TAC) for Standards and Performance Metrics
+from QED-C provided an open-source suite of quantum benchmarks
+([GitHub](https://github.com/SRI-International/QC-App-Oriented-Benchmarks)) to
+measure the effectiveness of quantum computing hardware for specific
+applications (further information provided in this
+[preprint](https://arxiv.org/abs/2110.03137)). This provided some of the first
+application-oriented benchmarks that we made public with supporting data along
```
Comment (nathanshammah) [inline posts/_markdown/metriq_qedc.md:19]:
```suggestion
application-oriented benchmarks that were made public with supporting data along
```

**Review by nathanshammah — CHANGES_REQUESTED** [review]
Looks great, thanks for the nice additions @vprusso regarding the Python client. You just need to update the index at blog.html and generate the html file from this md with the instructions from the Readme.

File: posts/_markdown/metriq_qedc.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title: Automatically running QED-C benchmarks on Metriq
+date: 23 Feb 2023
+author: Metriq Team
+---
+
+We are excited to announce the integration of an automated pipeline to add
+state-of-the-art benchmark from the Quantum Economic Development Consortium
+([QED-C](https://quantumconsortium.org/)) into **Metriq**, the open platform that
```
Comment (willzeng) [inline posts/_markdown/metriq_qedc.md:9]: Should be a hyperlink to metriq!

**Comment by willzeng** [comment]
Made one small comment but this looks great to ship from me!

**Comment by vprusso** [comment]
> d91beb9

Covered in [d91beb9](https://github.com/unitaryfund/unitary-fund/pull/300/commits/d91beb9e7e3502f1c5404cecde93a1e1c607b32f)

**Comment by vprusso** [comment]
> Made one small comment but this looks great to ship from me!

Covered in [d91beb9](https://github.com/unitaryfund/unitary-fund/pull/300/commits/d91beb9e7e3502f1c5404cecde93a1e1c607b32f)

**Comment by vprusso** [comment]
Addressed comments from @natestemen in [d91beb9](https://github.com/unitaryfund/unitary-fund/pull/300/commits/d91beb9e7e3502f1c5404cecde93a1e1c607b32f)

**Comment by nathanshammah** [comment]
@vprusso one small thing we missed: Can you please change the files names to 2023_metriq_qedc as it's our current format? Thanks.

File: posts/metriq_qedc.html [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+                                <p>We are excited to announce the integration of an automated pipeline to add state-of-the-art benchmark from the Quantum Economic Development Consortium (<a href="https://quantumconsortium.org/">QED-C</a>) into <strong>Metriq</strong>, the open platform that makes transparent, accessible benchmarks available to everyone in the quantum computing community.</p>
+                                <p>The Technical Advisory Committee (TAC) for Standards and Performance Metrics from QED-C provided an open-source suite of quantum benchmarks (<a href="https://github.com/SRI-International/QC-App-Oriented-Benchmarks">GitHub</a>) to measure the effectiveness of quantum computing hardware for specific applications (further information provided in this <a href="https://arxiv.org/abs/2110.03137">preprint</a>). This provided some of the first application-oriented benchmarks that were made public with supporting data along with a preprint–an incredibly valuable asset to the benchmarking community. Recently, the QED-C benchmarks suite was updated and new optimization benchmarks (<a href="https://arxiv.org/abs/2302.02278">preprint</a>).</p>
```
Comment (WrathfulSpatula) [inline posts/metriq_qedc.html:81]: Same comment as above: m dash (—)

File: posts/_markdown/metriq_qedc.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+accessible benchmarks available to everyone in the quantum computing community.
+
+The Technical Advisory Committee (TAC) for Standards and Performance Metrics
+from QED-C provided an open-source suite of quantum benchmarks
+([GitHub](https://github.com/SRI-International/QC-App-Oriented-Benchmarks)) to
+measure the effectiveness of quantum computing hardware for specific
+applications (further information provided in this
+[preprint](https://arxiv.org/abs/2110.03137)). This provided some of the first
+application-oriented benchmarks that were made public with supporting data along
+with a preprint–an incredibly valuable asset to the benchmarking community.
```
[the remark this answers is not in the document]
Comment (WrathfulSpatula) [reply posts/_markdown/metriq_qedc.md:20]: Alternatively, just make it a comma to indicate a parenthetical phrase.

[the remark this answers is not in the document]
Comment (vprusso) [reply posts/_markdown/metriq_qedc.md]: I loved this comment--great attention to detail! Fixed in [568509d](https://github.com/unitaryfund/unitary-fund/pull/300/commits/568509d2de2f168229634c6bd0eff87385a67a23)

**Comment by vprusso** [comment]
Updated and responded to all comments: [568509d](https://github.com/unitaryfund/unitary-fund/pull/300/commits/568509d2de2f168229634c6bd0eff87385a67a23)

File: blog.html [hunk 5]
```diff
@@ -72,6 +72,10 @@
               </p>
               <section class="light">
                 <ul style="list-style-type: none">
+                  <li class="leading-block">
+                    <span class="blog-post-date">23 February, 2023</span>
```
Comment (nathanshammah) [inline blog.html:76]:
```suggestion
                    <span class="blog-post-date">28 February, 2023</span>
```

File: posts/2023_metriq_qedc.html [hunk 6]
```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+<script src="https://kit.fontawesome.com/22ffc91bdd.js" crossorigin="anonymous"></script>
+
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8">
+    <meta name="date" content='2023-02-23'>
```
Comment (nathanshammah) [inline posts/2023_metriq_qedc.html:9]:
```suggestion
    <meta name="date" content='2023-02-28'>
```

File: posts/2023_metriq_qedc.html [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+                            </button>
+                        </div>
+                        <section class="light">
+
+
+                            <right>
+                                <h2 class="leading-arrow">Automatically running QED-C benchmarks on Metriq</h2>
+                                <h3></h3>
+                            </right>
+                            2023-02-23
```
Comment (nathanshammah) [inline posts/2023_metriq_qedc.html:75]:
```suggestion
                            2023-02-28
```

**Review by nathanshammah — APPROVED** [review]

**Comment by vprusso** [comment]
Awesome. Thank you for the updates, @nathanshammah !

**Comment by vprusso** [comment]
MWR! 🚀

**Pull request merged**